### ISSUE TITLE
cookiecutter doesn't parse ssh URLs

### ISSUE THREAD

**Issue by zenzic64** [audreyr/cookiecutter#145]
We have a local gitlab server and I tried running cookiecuter against a sample template repo I'd put on it.  It failed trying to open the cookiecutter.json file.  I eventually tracked it down to the way the URL argument is being parsed.  It assumes that the URL is an http URL, but we use ssh URLs to access the server. 

I put in the following kludge and it seems to work ok.  I'm sure there's a more elegant solution, but it's what I came up with:

```
cookiecutter/vcs.py
69c69,72
<     tail = os.path.split(repo_url)[1]
---
>     if repo_url.split('://')[0] in ['http', 'https']:
>         tail = os.path.split(repo_url)[1]
>     else:
>         tail = repo_url.rsplit(':')[1]
```

Thanks for a really great tool!

Jerry

**Comment by michaeljoseph** [comment]
Thoughts on using https://github.com/FriendCode/giturlparse.py to fix this and #122?
@zenzic64 @pydanny @BrainGrylls

**Comment by pydanny** [comment]
`giturlparse.py` looks like an awesome tool to resolve this issue. But...

`cancerhermit` is squatting the package on PyPI (https://github.com/FriendCode/giturlparse.py). Note that the link to the GitHub repo is not to the author of the package itself, but to a non-existent repo on GitHub. 

Until I hear otherwise from PyPI maintainers, I'm worried about using any package under their control. Pinging them now.

**Comment by michaeljoseph** [comment]
@pydanny https://pypi.python.org/pypi/giturlparse.py

**Comment by BrainGrylls** [comment]
A old PR (#122 ) of mine could fix the ssh location issue without having to add an additional dependency. 
``giturlparse.py`` on the other hand looks way more flexible...

**Comment by michaeljoseph** [comment]
Fixed by #490.
